test/methods.js

```diff
@@ -0,0 +1,300 @@
+var sinon = require('sinon');
+var should = require('should');
+var httpUtils = require('../lib/utils/httpUtils.js');
+var constants = require('../lib/utils/constants.js');
+var _ = require('underscore');
+var smartsheet = require('../index.js').createClient({accessToken: "token"});
+
+describe('Method Unit Tests', function () {
+    it("a test", function () {
+
+    });
```

Comment:
Is this needed to satisfy something in the test runner?

Reply:
Nope. I made that to verify that the mocha was finding my test file. I'll delete it